quiz.css

```diff
@@ -0,0 +1,9 @@
+.border {
+  border-width: 2px;
+  border-style: solid;
+}
```

Comment:
There are some odd layout issues when I edit a card. The height changes, which can cause the bottom card to jump left. If you add a fixed height to each card here, such as 250px, it should fix that.